src/Morphir/Visual/ViewValue.elm

```diff
@@ -120,7 +120,7 @@ viewValueByLanguageFeature config value =
                     ViewList.view config (viewValue config) itemType items
 
                 Value.Variable tpe name ->
-                    el []
+                    el [ width Element.fill, center]
```

Comment:
Let's be consistent and use `fill` instead of `Element.fill`.

Reply:
Comment addressed